src/models/Genre/getAll.ts

```diff
@@ -0,0 +1,29 @@
+import { DataWithPaginationResponse } from '../../types/apiResponse';
+import { cdaClient, CONTENTFUL_LIMIT } from '../../utils/contentful';
+import { Genre } from './GenreModel';
+import parseGenre from './parseGenre';
+
+export default async function getAll({ page = 1 }: { page?: number } = {}): Promise<
+	DataWithPaginationResponse<Genre>
+> {
+	const limit = CONTENTFUL_LIMIT;
```

Comment:
Should you make this configurable so people can select how many items they want per page?